lib/advanced-open-file-view.coffee

```diff
@@ -323,8 +325,15 @@ class AdvancedFileView extends View
   undo: ->
     if @pathHistory.length > 0
       @miniEditor.setText @pathHistory.pop()
-    else
-      atom.beep()
+      @miniEditor.getModel().scrollToCursorPosition()
```

Comment:
Good idea to scroll to the cursor after an undo! That's standard behavior that we're missing.